### PR TITLE
Install lsof pkg if not installed

### PR DESCRIPTION
**Pull request by yafu-1**
Signed-off-by: Yan Fu <yafu@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/domain/virsh_dump.py [hunk 1]
```diff
@@ -180,6 +180,14 @@ def check_dump_format(dump_image_format, dump_file):
     # Configure dump_image_format in /etc/libvirt/qemu.conf.
     qemu_config = utils_config.LibvirtQemuConfig()
     libvirtd = utils_libvirtd.Libvirtd()
+
+    # Install lsof pkg if not installed
+    cmd = 'rpm -q lsof || yum -y install lsof'
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_dump.py]: I assume utils_package.package_install can satisfy your request.

Comment (yafu-1) [reply libvirt/tests/src/virsh_cmd/domain/virsh_dump.py]: Done.

File: libvirt/tests/src/virsh_cmd/domain/virsh_dump.py [hunk 2]
```diff
@@ -180,6 +182,11 @@ def check_dump_format(dump_image_format, dump_file):
     # Configure dump_image_format in /etc/libvirt/qemu.conf.
     qemu_config = utils_config.LibvirtQemuConfig()
     libvirtd = utils_libvirtd.Libvirtd()
+
+    # Install lsof pkg if not installed
+    if not utils_package.package_install("lsof"):
+        test.error("Failed to install lsof in host\n")
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/domain/virsh_dump.py]: this should be `test.cancel`

Comment (yafu-1) [reply libvirt/tests/src/virsh_cmd/domain/virsh_dump.py]: Done.

**Pull request merged**